src/assets/translations/fr/main.json

```diff
@@ -1443,6 +1448,7 @@
         "sell": "Vendre",
         "trade": "Échanger"
       },
+      "equity": "Fonds propres",
```

Comment:
Hmm not sure about this one, that's supposed to represent all fund across wallet/DeFi opportunities, in other words delegated too, not just owned

Reply:
Welp, if you want to change it we'll do it in the weekly PR ;)